Basic/Invaders/Assets/Scripts/PlayerBullet.cs

```diff
@@ -14,6 +13,8 @@ public class PlayerBullet : MonoBehaviour
 
     private void Update()
     {
+        if (!NetworkManager.Singleton.IsServer) return;
```

Comment:
I don't understand if we are not a Server we return since we don't want the client to update the bullets since they are NetworkTransform. So we are actually returning.